lld/wasm/Driver.cpp

```diff
@@ -375,6 +378,16 @@ void LinkerDriver::createFiles(opt::InputArgList &args) {
     case OPT_no_whole_archive:
       inWholeArchive = false;
       break;
+    case OPT_start_lib:
+      if (inLib)
+        error("nested --start-lib");
+      inLib = true;
+      break;
+    case OPT_end_lib:
+      if (!inLib)
+        error("stray --end-lib");
```

Comment:
```suggestion
        error("--end-lib without matching --start-lib");
```

Reply:
These match the ELF errors, so maybe we should leave them the same for now?  And maybe followup by making them more clear?

Reply:
Yeah, that's fine. Too bad about all this duplication :)